### PR TITLE
Add CI script to add TXT record to DigitalOcean

### PR DESCRIPTION
**Pull request by benhylau**
Tested on #132 https://travis-ci.org/ournetworks/ournetworks.ca/builds/579303878#L525

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -57,6 +58,7 @@ after_deploy:
   - rm -fr .dat .well-known
   - make-relative https://ournetworks.ca
   - ipd . -O -C
+  - bash scripts/txt.sh ournetworks.ca ipfs-website "ipfs/<hash>"
```
Comment (garrying) [inline .travis.yml]:
```suggestion
  - bash scripts/txt.sh ournetworks.ca ipfs-website "dnslink=/ipfs/<hash>"
```
ref: http://dnslink.io/

File: scripts/txt.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Update TXT record for a domain managed by Digital Ocean.
+#
+# Usage:
+#
+#     export DO_TOKEN=<do_token>
+#     ./txt.sh <domain> <name> <value>
+#
+# Example:
+#
+#     ./txt.sh ournetworks.ca ipfs-website "ipfs/QmQQfD1ztg1aG82VmUmutusbxn7tL4c2o1qHv1ivyWbNj6"
```
Comment (garrying) [inline scripts/txt.sh]:
```suggestion
#     ./txt.sh ournetworks.ca ipfs-website "dnslink=/ipfs/QmbfXYeieYgXZs7KdomE5wHQwoLRCPkYywwWpo1C9gfC3f"
```

File: .travis.yml (hunk 1 above)
Comment (garrying) [reply .travis.yml]: Would it make sense to change the `TXT` hostname to `_dnslink.ournetworks.ca`? Following what's on dnslink.io

**Comment by benhylau** [comment]
@garrying is this still needed? Or leave it here until the ipfs publish hash issue is resolved?

**Pull request closed without merging**